__tests__/components/AmountInput.spec.ts

```diff
@@ -149,4 +156,83 @@ describe('AmountInput', () => {
         const rightResultMaxRelativeAmount = await validate('89999999990', rule);
         expect(rightResultMaxRelativeAmount.valid).toBeTruthy();
     });
+
+    test('use maximum balance when the selected account is regular', async () => {
+        // Arrange:
+        const maxBalance = '15';
```

Comment:
It comes from the `mosaicsMock` array defined here:

https://github.com/symbol/desktop-wallet/blob/cf1f811506bb5b898e386b70309daf29ac2080a4/__mocks__/mosaics.ts#L7

I know it's subtle, so instead of that should we refactor it to the following? 🤔 
```
const selectedMosaic = mosaicsMock[0];
const maxBalance = (selectedMosaic.balance / Math.pow(10, selectedMosaic.divisibility)).toString();
```